--- jit/CMakeLists.txt	2021-12-14 18:41:20.844000000 +0600
+++ jit/CMakeLists.txt.new	2021-12-13 16:28:45.163475144 +0600
@@ -519,7 +519,7 @@
 set(JIT_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
 # Creates a static library "clrjit_static" to link into the VM.
-add_subdirectory(static)
+#add_subdirectory(static)
 
 if (CLR_CMAKE_TARGET_OSX AND CLR_CMAKE_TARGET_ARCH_ARM64)
   set(TARGET_OS_NAME unix_osx) # Apple Arm64 has a special ABI, distinguish it.
